_docs/payout-details.md

```diff
@@ -423,3 +423,114 @@ BJ
 <div class="alert alert-info" markdown="1">
 **Note** `XOF::Bank` payouts are currently in beta phase. At this time, we offer payouts to accounts in Senegal and Benin only.
 </div>
+
+# SouthAfrica
+
+## ZAR::Bank
+
+For South African bank payments please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "street": "Main Street",
+  "postal_code": "AB0001",
+  "city": "Cape Town",
+  "bank_code": "334810",
+  "bank_account": "12345678",
+  "bank_name": "The Bank Name", // Not required if the bank is included in the list below
+  "phone_number": "+27119785313",
+  "entity_type": "6", // Not required if entity type is person
+  "transfer_reason_code": "185"
+}
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="zar-bank-details" raw=data-raw %}
+
+List of banks and corresponding `bank_code` are:
+
+{% capture data-raw %}
+```
+Standard Bank: 051001
+First National Bank: 250655
+ABSA: 632005
+Nedbank: 198765
+Investec: 580105
+Capitec Bank: 470010
+Bank of Athens: 410506
+Bidvest Bank: 462005
+African Bank: 430000
+Mercantile Bank: 450905
+SA Post Office: 460005
+Tyme Bank: 678910
+Ubank: 431010
+Discovery Bank: 679000
+Bank Zero: 888000
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="zar-bank-codes" raw=data-raw %}
+
+List of transfer reasons and corresponding `transfer_reason_code` are:
+
+{% capture data-raw %}
+```
+Sending money into my own account: 183
+Donations and gifts: 184
+Sending money to a friend, family member or any third party person: 185
+Mortgage repayments: 186
+Business Travel Payments: 187
+Personal Travel Payments: 188
+Tuition fees: 189
+Investment into property by a foreign individual: 192
+Investment by a foreign individual - other: 193
+Legal services: 196
+Accounting services: 197
+Management consulting services: 198
+Advertising & market research services: 200
+Managerial services: 201
+Cultural and recreational services: 205
+Salary paid to South African Resident Temporarily Abroad: 206
+Salary paid to a non-resident employee in South Africa: 207
+Salary paid to a foreign national contract worker in South Africa: 208
+Pensions: 213
+Annuities: 214
+Inheritances: 215
+Alimony: 216
+Tax - Income tax: 217
+Tax - VAT refunds: 218
+Tax - Other: 219
+Dividends: 222
+Commission or brokerage: 224
+Rental: 225
+Income earned abroad by a resident on an individual investment: 226
+Architectural, engineering and other technical services: 245
+Payments for data, news related and news agency fees: 249
+Computer-related services including maintenance, repair and consultancy: 279
+Proceeds for other business services not included elsewhere: 309
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="zar-transfer-reason-codes" raw=data-raw %}
+
+List of entity types and corresponding `entity_type` codes are:
```

Comment:
Let's skip this for now - we have a separate epic to figure out how we'll support business recipients